Aspect of the hawk triggers Error

Whenever I use Aspect of the Hawk I get an error saying

[ERROR] [Outfitter] No events found for Hawk

I am a fan of the add-on, but this error is very obnoxious.

Thank you

--- a/Outfitter.lua
+++ b/Outfitter.lua
@@ -954,6 +954,7 @@ Outfitter.cSpecialIDEvents =
 	GhostWolf = {Equip = "GHOST_WOLF", Unequip = "NOT_GHOST_WOLF"},
 
 	Feigning = {Equip = "FEIGN_DEATH", Unequip = "NOT_FEIGN_DEATH"},
+	Hawk = {Equip = "HAWK", Unequip = "NOT_HAWK"},
 	
 	Evocate = {Equip = "EVOCATE", Unequip = "NOT_EVOCATE"},
 	

